src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/AccessorBuilder.cs

```diff
@@ -23,12 +25,18 @@ internal static class FastInvokerBuilder
         private static readonly MethodInfo s_createSetterInternal = typeof(FastInvokerBuilder).GetMethod(nameof(CreateSetterInternal), BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Static)!;
         private static readonly MethodInfo s_make = typeof(FastInvokerBuilder).GetMethod(nameof(Make), BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Static)!;
 
-        public static Func<object> GetMakeNewInstanceFunc(Type type)
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2060:MakeGenericMethod",
+            Justification = "The call to MakeGenericMethod is safe due to the fact that FastInvokerBuilder.Make<T> is not annotated.")]
```

Comment:
Tehnically, `FastInvokerBuilder.Make<T>` is annotated because it has `where T : new()`. So this should say that we are preserving constructors to make it OK.